src/main/java/com/minecolonies/util/StructureWrapper.java

```diff
@@ -756,42 +661,42 @@ public Item getItem()
     }
 
     /**
-     * @return The name of the schematic.
+     * @return The name of the structure.
      */
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)